docs/user/environment-variables.rst

```diff
@@ -50,6 +50,12 @@ Read the Docs builders set the following environment variables automatically for
 
     :Examples: ``en``, ``it``, ``de_AT``, ``es``, ``pt_BR``
 
+.. envvar:: READTHEDOCS_VIRTUALENV_PATH
+
+    Path where Read the Docs created the virtualenv for this build.
```

Comment:
```suggestion
    Path for the virtualenv that was created for this build.
    Only exists for builds using Virtualenv and not Conda. 
```

Can we link the virtualenv link somewhere useful? Perhaps https://docs.readthedocs.io/en/stable/builds.html#understanding-what-s-going-on ?